src/utilities/transform-mdx.tsx

```diff
@@ -1,32 +1,53 @@
 import * as Components from '@atom-learning/components'
 import { Heading, Link, List, Text } from '@atom-learning/components'
+import theme from '@atom-learning/theme'
 import hydrate from 'next-mdx-remote/hydrate'
 import renderToString from 'next-mdx-remote/render-to-string'
 import { MdxRemote } from 'next-mdx-remote/types'
 import * as React from 'react'
 
-import { CodeBlock, Divider, InlineCode } from '../components'
+import {
+  CodeBlock,
+  ColorPalette,
+  Divider,
+  InlineCode,
+  Scale
+} from '../components'
 
 const components: MdxRemote.Components = {
   h2: (props) => <Heading {...props} as="h2" css={{ mt: '$5', mb: '$4' }} />,
-  h3: (props) => (
-    <Heading {...props} as="h3" size="sm" css={{ mt: '$4', mb: '$3' }} />
+  h3: (props) => <Heading {...props} as="h3" size="sm" css={{ my: '$4' }} />,
+  h4: (props) => (
+    <Heading {...props} as="h4" size="xs" css={{ mt: '$4', mb: '$3' }} />
   ),
   p: (props) => <Text {...props} css={{ mb: '$4' }} />,
   ul: (props) => <List {...props} css={{ mb: '$4' }} />,
-  li: (props) => (
-    <List.Item {...props} css={{ '&:not(:last-child)': { mb: '$2' } }} />
-  ),
+  li: List.Item,
   inlineCode: InlineCode,
+  blockquote: (props) => (
+    <Text
+      {...props}
+      as="blockquote"
+      css={{ pl: '$4', my: '$4', color: '$tonal600' }}
+    >
+      {props.children.props.children}
```

Comment:
This is weird but a few things to note:
- MDX will always transform `> a blockquote` into `<blockquote><p></p></blockquote>` and render our specific styles for `p` which I want to avoid, `<blockquote></blockquote>` is fine
- It will therefore always props of `children` (which is `p`) that then has `children` of a string, so `props.children` to get the `p` and then `props.children` of that to get the text

Reply:
🤯